packages/client/src/views/Home/RecordAudit.tsx

```diff
@@ -231,6 +234,29 @@ const ShowOnMobile = styled.div`
     margin-top: 32px;
   }
 `
+const PaginationDiv = styled.div`
```

Comment:
If RecordAudit cant use pagination at the moment because of the download issue.  Should pagination be removed from this for now?